README.md

```diff
@@ -146,6 +155,30 @@ let x = 2;
 let y = 1 * _x;
 ```
 
+### Conditionals
+
+Conditional statements allow a block of code to be executed if a condition is true or if specified, execution of an alternative block. 
```

Comment:
This sentence could be better worded I think.

Reply:
yeah, you're right.,
I think I will remove it, no other section in my README has this kind of explanation.